src/Tests/Microsoft.NET.Publish.Tests/GivenThatWeWantToPublishAnAotApp.cs

```diff
@@ -705,6 +705,34 @@ public void It_publishes_with_implicit_rid_with_NativeAotApp(string targetFramew
             }
         }
 
+        [RequiresMSBuildVersionTheory("17.0.0.32901")]
+        [InlineData(ToolsetInfo.CurrentTargetFramework)]
+        public void It_builds_with_dynamiccodesupport_false(string targetFramework)
+        {
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows) || RuntimeInformation.IsOSPlatform(OSPlatform.Linux))
```

Comment:
Is it a pattern to restrict to these platforms?

Reply:
The rest of the tests in this file do, presumably because NativeAot isn't supported on macOS.

I can remove this check since we are just building the project here.